### PR TITLE
Add Mint installation instructions

### PR DESCRIPTION
**Pull request by yonaskolb**
This adds [Mint](https://github.com/yonaskolb/Mint) installation instructions to the readme 😄

### CODE REVIEW

**Comment by jpsim** [comment]
**tl;dr; please move this to after the CocoaPods section**

Thanks for the PR! I think Mint is awesome, but there are a few reasons why I still want to lead people to Homebrew first, then CocoaPods, then Mint.

First, homebrew is still the canonical way to install CLI tools on macOS. It has the added advantage of not requiring to build from source, thus speeding up the process, since Homebrew just downloads the bottled version of the formula. It's also the installation method that we've supported the longest, so it's well tested.

Second, CocoaPods since many people are _already_ using it. It also avoids compiling the binary upon installation, simply downloading it from GitHub releases. It also helps version-lock SwiftLint to projects by specifying the version[s] in the Podfile.

**Comment by yonaskolb** [comment]
Yeah of course, makes sense! Sorry, that was presumptuous of me to add it at the top, I didn't really think about it.
I've moved it to after cocoapods 👍

**Comment by jpsim** [comment]
Thanks!
